es6-promises/index.js

```diff
@@ -0,0 +1,10 @@
+const takeAChance = require('./take-a-chance');
+const promiseObject = takeAChance('Francis');
+
+promiseObject.then(value => {
+  console.log(value);
+});
+
+promiseObject.catch(Error => {
+  console.error(Error.message);
+});
```

Comment:
It's a bad idea to capitalize the `catch` param as `Error`, since it then prevents any use of the global `Error` constructor: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error